google/cloud/storage/internal/signed_url_requests.cc

```diff
@@ -147,6 +147,14 @@ void V4SignUrlRequest::SetOption(VirtualHostname const& hostname) {
   virtual_host_name_ = hostname.has_value() && hostname.value();
 }
 
+void V4SignUrlRequest::SetOption(DomainNamedBucket const& o) {
```

Comment:
can this follow the expected name `BucketBoundHostname`

Reply:
Both mean nothing to me, so 🤷‍♂ .  @dopiera can you modify this PR to include that change or would a future PR be better?